mkdocs.yml

```diff
@@ -54,6 +54,7 @@ nav:
     - 'Document Payloads': payloads.md
     - 'Spelling Correction': Spellcheck.md
     - 'Phonetic Matching': Phonetic_Matching.md
+    - 'Upgrade to 2.0': Upgrade_to_2.0.md
```

Comment:
@MeirShpilraien I believe this belongs under administration.  It's not about the actual usage